roles/gitops/rendering-apps-files/templates/console/Chart.yaml.j2

```
@@ -0,0 +1,13 @@
+apiVersion: v2
+type: application
+name: console
+version: 1.0.0
+dependencies:
+- name: cpn-cnpg
+  alias: cpn-cnpg
+  version: {{ dsc.cpnCnpg.chartVersion }}
+  repository: {{ dsc.cpnCnpg.helmRepoUrl }}
+- alias: console
+  name: cpn-console
+  repository: {{ dsc.console.helmRepoUrl }}
+  version: {{ dsc.console.chartVersion | quote }}
```

```suggestion
  version: {{ dsc.console.chartVersion }}
```
Si on utilise le même formalisme que pour cpnCnpg dans la DSC on devrait avoir le même rendu ici sans besoin des quotes

---

Bonus : ajouter une ligne vide en fin  de fichier, comme suggéré par Github avec son :no_entry:

---

En fait, c'est à cause de la notation `">=2.0.0 <3.0.0"` (https://github.com/cloud-pi-native/socle/blob/feat/gitops-console-deployment6/roles/socle-config/files/releases.yaml#L21) nécessaire (parce qu'on ne fait pas pareil sur Argo) pour avoir le même effet que le `2.*.*` qu'on a actuellement.